frontend/components/site/siteGithubBranchesTable.js

```diff
@@ -1,46 +1,31 @@
 import React from 'react';
 
 import LoadingIndicator from '../loadingIndicator';
+import BranchViewLink from '../branchViewLink';
 import { SITE, GITHUB_BRANCHES } from '../../propTypes';
 
-const tableHeader = () => (
```

Comment:
I got rid of some of these functions and just moved the JSX markup into the `renderTable` method because I didn't feel like all the functions were increasing clarity/readability.